requirements/test.in

```diff
@@ -38,4 +38,6 @@ tritonclient==2.51.0
 
 numpy < 2.0.0
 runai-model-streamer==0.11.0
-runai-model-streamer-s3==0.11.0
\ No newline at end of file
+runai-model-streamer-s3==0.11.0
+
+flashinfer-python==0.2.3 # required for flashinfer tests
```

Comment:
this is installed in the docker. we need to build it. the pypi version is just source code. we cannot install it directly from pypi.